extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -38,6 +40,17 @@ const TAG = 'amp-bind';
 const AMP_CSS_RE = /^(i?-)?amp(html)?-/;
 
 /**
+ * Tags under which bind should observe mutaitons to detect added/removed
+ * bindings.
+ * @type {!Object<string, boolean>}
+ * @private
+ */
+const DYNAMIC_TAGS = map({
+  'TEMPLATE': true,
+  'AMP-LIVE-LIST': true,
```

Comment:
@cramforce 

:+1:, I'm in the middle of implementing a similar alternative strategy [here](https://github.com/ampproject/amphtml/pull/8367)